var/spack/repos/builtin/packages/perl-html-parser/package.py

```diff
@@ -9,7 +9,7 @@
 class PerlHtmlParser(PerlPackage):
     """HTML parser class"""
 
-    homepage = "http://search.cpan.org/~gaas/HTML-Parser-3.72/Parser.pm"
+    homepage = "https://metacpan.org/pod/HTML::Parser"
     url      = "http://search.cpan.org/CPAN/authors/id/G/GA/GAAS/HTML-Parser-3.72.tar.gz"
```

Comment:
I assume that if search.cpan.org is discouraged for homepages, it's also discouraged for URLs (even if repology doesn't test it). We should add fetching support like we did for `pypi` at some point. This can happen in a separate PR though.

Reply:
yeah that's a good idea!